### ISSUE TITLE
support for alias value of false like the browser field, maps to empty module

### ISSUE THREAD

**Issue by jeffbski** [benbria/aliasify#35]
It would be nice if aliasify would also support simply setting an alias value to false indicating that an empty module would need to be used as the replacement.

```json
{
    "aliasify": {
        "aliases": {
            "crypto": false
        }
    }
}
```

This would cause crypto to be replaced with a require to an empty module just like browserify handles the false for its browser field.

Without this change, one needs to specify this:

```json
{
    "aliasify": {
        "aliases": {
            "crypto": "browserify/lib/_empty"
        }
    }
}
```

With this change, it is more intuitive to the user since it works like browser field and we aren't exposing a path browserify's empty which is private. We can keep the path internal to aliasify and adjust as necessary or even just include our own empty file which we point to (maybe that is even better).

It is a pretty common need to stub out requires for the browser so making this easy is a big win.

I would have submitted a PR since this probably just an extra if statement, but my coffeescript skills are pretty lacking.

Thanks for a great module!

**Comment by jwalton** [comment]
Fixed, and published as v1.9.0.  :)

**Comment by jeffbski** [comment]
Excellent. Thanks!
